pkg/influx/api.go

```diff
@@ -59,9 +69,8 @@ func (a *API) handleSeriesPush(w http.ResponseWriter, r *http.Request) {
 		a.handleError(w, r, err)
 		return
 	}
-
 	a.recorder.measureMetricsWritten(len(rwReq.Timeseries))
-	_ = level.Debug(a.logger).Log("msg", "successful series write", "len", len(rwReq.Timeseries))
+	level.Debug(a.logger).Log("msg", "successful series write", "len", len(rwReq.Timeseries))
```

Comment:
this is going to be a very noisy line, I think we should only record the metric and not also print